source/SH3/graphics/texture.cpp

```diff
@@ -23,12 +23,21 @@
 
 using namespace sh3_graphics;
 
-#pragma pack(push, 1)
```

Comment:
Actually, thinking about it, will the structures have any padding at all? I know that `char` is required to have an alignment of 1, I'm guessing that `std::uint8_t` should be the same. We could just `static_assert` that in the `color.hpp` header and be done with it.

Reply:
Hmm, I neglected to think of that actually. I'll try it out and push it up if it works. Packing structs is a bit of a nasty habit I have from uni sorry.